src/settings/BatchGroupConfigurationSettings/BatchVouchers/BatchVouchersContainer.js

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+
+import { batchVouchersFromPropResource } from '../../../common/resources';
+import BatchVouchers from './BatchVouchers';
+
+const BatchVouchersContainer = ({ mutator, batchVoucherId }) => {
+  const downloadBatchVouchers = useCallback(
+    () => {
+      mutator.batchVouchers.GET();
+      // then download file
+    },
+    [mutator.batchVouchers],
+  );
+
+  return (
+    <BatchVouchers
+      downloadBatchVouchers={batchVoucherId ? downloadBatchVouchers : undefined}
```

Comment:
IMHO, you can follow the common pattern and put it above, something like `if (!batchVoucherId ) return null;` or make ternary in return itself